tools/update-lib-index.js

```diff
@@ -52,6 +46,33 @@ module.exports = {
     }
   }
 }
+
+const baseConfig = {plugins: {vue: mod}}
+
+mod.configs = {
```

Comment:
I'm not sure why you first define `mod` and then set its `configs` property. Can we inline it again?

Reply:
it's to ref the mod: `plugins: {vue: mod}`.  
you can see it in the eslint migration docs: https://eslint.org/docs/head/extend/plugin-migration-flat-config#migrating-configs-for-flat-config :)

Reply:
Ah, I see. Can we rename the variable to `plugin` then, please?

Reply:
https://github.com/vuejs/eslint-plugin-vue/pull/2407/commits/ae60a4af72728d2ed45ff610977a3c55f49f0693